R/get_info.R

```diff
@@ -0,0 +1,37 @@
+# get_info ---------------------------------------------------------
+#' Get information on the model
+#'
+#' @param exp Experiment object
+#' @param pattern can be either 'Parameters' or 'Outputs' for parameters and
+#'        observation rates, respectively.
+#' @param type can be 'name' for parameter and observation rate names, '
+#'        type' for parameter types.
+#'
+#' @return A vector of parameter or variable names,
+#'
+#' @export
+#'
+#' @example inst/examples/get_info.R
+#' @rdname get_info
+#' @export
+get_info <- function(exp, pattern, type){
+  model <- model(exp)
+  query <- model[["info"]][[pattern]]
+  if(!is.list(query))
+    query <- list(query)
+  out <- unlist(lapply(query, function(x) x[[type]]))
+  names <- type
```

Comment:
The `if` seems to be only necessary when calling: 
```
get_info(exp, ".attrs", "experiment")
```
But I don't know in which case it will be necessary to do that as the user can do: 
```
attributes(exp)
```
to have access to the attributes of the experiment object.
So I agree to remove this line and also the line ` names <- type` if everybody agrees.

Reply:
In case of calling 

`
get_info(exp, ".attrs", "experiment")
`
the line `names <- type` is needed. I can add brackets to the code to be clearer then.